public/components/register_model/register_model.tsx

```diff
@@ -151,13 +152,46 @@ export const RegisterModelForm = () => {
         component="form"
       >
         <EuiPanel>
-          <EuiPageHeader pageTitle="Register Model" />
-          <EuiText style={{ maxWidth: 420 }}>
-            <small>
-              Register your model to collaboratively manage its life cycle, and facilitate model
-              discovery across your organization.
-            </small>
-          </EuiText>
+          {latestVersionId && (
+            <div>
+              <EuiPageHeader pageTitle="Register version" />
+              <EuiText style={{ maxWidth: 420 }}>
+                <small>
+                  Register a new version of Image-classifiar.The version number will be
+                  automatically incremented. For more information on versioning, see{' '}
+                  <EuiLink href="#" external>
+                    Model Registry Documentation
+                  </EuiLink>
+                  .
+                </small>
+              </EuiText>
+            </div>
+          )}
+          {formType === 'import' && !latestVersionId && (
+            <div>
+              <EuiPageHeader pageTitle="Register model" />
+              <EuiText style={{ maxWidth: 420 }}>
+                <small>
+                  Register a pre-trained model. For more information, see{' '}
+                  <EuiLink href="#" external>
+                    OpenSearch model repository documentation
+                  </EuiLink>
+                  .
+                </small>
+              </EuiText>
+            </div>
+          )}
+          {formType === 'upload' && !latestVersionId && (
+            <div>
+              <EuiPageHeader pageTitle="Register model" />
+              <EuiText style={{ maxWidth: 420 }}>
+                <small>
+                  Register your model to collaboratively manage its life cycle, and facilitate model
+                  discovery across your organization.
+                </small>
+              </EuiText>
+            </div>
+          )}
```

Comment:
Could we share the same structure? Like:
```tsx<>
  <EuiPageHeader
    pageTitle={latestVersionId ? 'Register version' : 'Register model'}
  />
  <EuiText style={{ maxWidth: 420 }}>
    <small>
    {latestVersionId && (
      <>
        Register a new version of Image-classifiar.The version number will be
        automatically incremented. For more information on versioning, see{' '}
        <EuiLink href="#" external>
          Model Registry Documentation
        </EuiLink>
        .
      </>
    )}
    {formType === 'import' && !latestVersionId && (
      
      <>
      Register a pre-trained model. For more information, see{' '}
      <EuiLink href="#" external>
        OpenSearch model repository documentation
      </EuiLink>
      .
    </>
    )}
    {formType === 'upload' && !latestVersionId && (
      <>
      Register your model to collaboratively manage its life cycle, and facilitate model
      discovery across your organization.
    </>
    )}
    </small>
  </EuiText>
</>;

```

Reply:
ok